src/renderer/utils/numbers.ts

```diff
@@ -27,3 +27,23 @@ export const numberFormatter = (value: any, unit = '', fractionDigits = 2) => {
 
     return `${formatter.format(parsedValue)}${unit}`;
 };
+
+
+export const toHex = (num: number): string => {
+    if (num < 0) {
+        return toHex(0xffffffff + num + 1);
+    }
+    return `0x${num.toString(16).toUpperCase()}`;
+};
+
+export const formatSize = (number: number): string => {
+    return new Intl.NumberFormat('en-US').format(number);
+};
+
+export const prettyPrintAddress = (address: number | null, memorySize: number): string => {
+    if (address === null) {
+        return '0'.padStart(memorySize.toString().length, '0');
+    }
+
+    return address.toString().padStart(memorySize.toString().length, '0');
+};
```

Comment:
Is this a leftover maybe?

Reply:
no

Reply:
What about renaming that function then and making it explicit that the formatting is for decimal numbers?